lib/ansible/galaxy/token.py

```diff
@@ -68,8 +68,16 @@ def __init__(self, access_token=None, auth_url=None, validate_certs=True, client
 
     @property
     def expired(self):
-        issue_time = datetime.strptime(self._issued_at, ISO_8601_FORMAT_STR)
-        almost_expiration = issue_time + timedelta(seconds=self._expires_in - 3)  # expire a few seconds before to handle proactively
+        if None in (self._issued_at, self._expires_in):
```

Comment:
I probably should update the existing unit tests instead of this...

Reply:
Also, I'm not sure if the "contains" check uses the identity (`is`) or value comparison (`==`) to match the elements internally. So there might be some unexpected behavior hidden behind such a check.